config/lti.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Meh, not super crazy about this ... not super crazy about the alternative either (hard-coding server names)

Reply:
Took me a second to figure out how this worked. Neither alternative seems great, I agree. Maybe just drop a comment here explaining how this works? Something like (If I'm understanding this correctly) "The deployment process drops a list of LTI servers into the secretes file, each of which needs a YAML mapping entry, so we create those here for consumption by the..." etc.

Reply:
Added a comment below about how things interact.